src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/VBDefineConstantsEncoding.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.VisualStudio.ProjectSystem.Debug
 
     [Export(typeof(INameValuePairListEncoding))]
     [ExportMetadata("Encoding", "NameQuotedValuePairListEncoding")]
```

Comment:
Does string here change because class name has changed?

Reply:
Yes, and the same symbol needs to change in the rule file.